packages/charts/package.json

```diff
@@ -103,6 +102,7 @@
     },
     "scripts": {
         "assemble": "run-s prettier-check lint build-lib:locl build-lib:prod compodoc-custom-theme build-examples:prod",
+        "assemble-dev": "yarn run build-lib:prod",
```

Comment:
for "assemble-dev" would be better to run build-lib with non prod config

Reply:
the intention is to have the same output as regular assemble without running too many things repeatedly (to be used when working on some feature and testing it's use in some linked project)

Reply:
but as long as dist-copy is part of build-lib:prod I can just remove it and use that one, I wanted to keep the dist-copy only as part of assemble scripts but I guess I will leave it as it was in the end